### PR TITLE
The plan for EF8

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

File: entity-framework/core/what-is-new/ef-core-8.0/plan.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+EF8 will align with .NET 8 as a long-term support (LTS) release. See the [.NET support policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) for more information.
+
+### Breaking changes
+
+EF8 will contain a small number of [breaking changes](xref:core/what-is-new/ef-core-8.0/breaking-changes) as we continue to evolve both EF Core and the .NET platform. Our goal is to minimize breaking changes as much as possible without stagnating the platform.
+
+## Themes
+
+Large investments for EF8 amd data access in .NET 8 fall under the following themes:
```
Comment (maumar) [inline entity-framework/core/what-is-new/ef-core-8.0/plan.md:34]: typo (amd)

File: entity-framework/core/what-is-new/ef-core-8.0/plan.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+> [!IMPORTANT]
+> This plan is not a commitment; it will evolve as we continue to learn throughout the release. Some things not currently planned for EF8 may get pulled in. Some things currently planned for EF8 may get punted out.
+
+## General information
+
+EF Core 8 is the next release after EF Core 7 and is scheduled for release in November 2023, at the same time as .NET 8. There are no plans for an EF Core 7.1 release.
+
+### Supported platforms
+
+EF8 currently targets .NET 6. This will likely be updated to .NET 8 as we near the release. EF8 does not target any .NET Standard version; for more information see [the future of .NET Standard](https://devblogs.microsoft.com/dotnet/the-future-of-net-standard/). EF8 will not run on .NET Framework.
```
Comment (ErikEJ) [inline entity-framework/core/what-is-new/ef-core-8.0/plan.md:24]: No .NET Framework - is that relevant any more?

Comment (ajcvickers) [reply entity-framework/core/what-is-new/ef-core-8.0/plan.md:24]: Better to explicitly state it just so it's 100% clear.

File: entity-framework/core/what-is-new/ef-core-8.0/plan.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+## Theme: Developer experience
+
+Creating a great experience for developers has always been a primary driving force for the team. Almost everything in the themes outlined here above relates to this in some way. In addition, a large part of the work planned for EF8 involves improving the developer experience in many small ways across multiple areas.
+
+All the issues that we may potentially work on for EF8 are [tracked on GitHub by the 'consider-for-current-release' label](https://github.com/dotnet/efcore/issues?q=is%3Aopen+is%3Aissue+label%3Aconsider-for-current-release). We will reassess these issues throughout the release and move issues into the [8.0 milestone](https://github.com/dotnet/efcore/issues?q=is%3Aopen+is%3Aissue+milestone%3A8.0.0) as we commit to working on them. Make sure to vote for issues that are important to you (👍) so that we can use this information when deciding with issues to work on.
```
Comment (maumar) [inline entity-framework/core/what-is-new/ef-core-8.0/plan.md:133]: typo? (with issues to work on -> which issues to work on)

**Comment by ajcvickers** [comment]
Updated.

**Review by maumar — APPROVED** [review]

**Pull request merged**